### PR TITLE
Update for Fortress: common, gui, msgs, physics and plugin

### PR DESCRIPTION
**Pull request by j-rivero**
Follow up problems with dependencies after #152 and #165. I'm pretty sure that we need common4, msgs8 and sensors bumps to make the whole fortress collection to install.

### CODE REVIEW

File: config/ignition_fortress_ubuntu_jammy.yaml [hunk 1]
```diff
@@ -68,12 +68,12 @@ filter_formula: "\
  Package (= libignition-physics5-tpe-dev) |\
  Package (= libignition-physics5-tpelib) |\
  Package (= libignition-physics5-tpelib-dev) \
-), ($Version (% 5.1.0+osrf-2*) | $Version (% 5.1.0+osrf-3*)) |\
+), ($Version (% 5.2.0-1*) | $Version (% 5.1.0+osrf-3*)) |\
```
Comment (chapulina) [inline config/ignition_fortress_ubuntu_jammy.yaml:71]: Should we remove the alternative version?

```suggestion
), $Version (% 5.2.0-1*) |\
```

File: config/ignition_fortress_ubuntu_jammy.yaml [hunk 2]
```diff
@@ -68,12 +68,12 @@ filter_formula: "\
...
 (Package (= ignition-plugin) |\
  Package (= libignition-plugin) |\
  Package (= libignition-plugin-dbg) |\
  Package (= libignition-plugin-dev) \
-), $Version (% 1.2.1+osrf-1*) |\
+), $Version (% 1.3.1-1*) |\
```
Comment (chapulina) [inline config/ignition_fortress_ubuntu_jammy.yaml:76]: I only see 1.3.0-1: http://packages.osrfoundation.org/gazebo/ubuntu-stable/pool/main/i/ignition-plugin/

```suggestion
), $Version (% 1.3.0-1*) |\
```

image

File: config/ignition_fortress_ubuntu_jammy.yaml [hunk 3]
```diff
@@ -68,12 +68,12 @@ filter_formula: "\
...
 (Package (= ignition-rendering6) |\
```
Comment (chapulina) [inline config/ignition_fortress_ubuntu_jammy.yaml:77]: Rendering also needs bumping:

image

**Review by chapulina — CHANGES_REQUESTED** [review]
Tools can be bumped:

image

File: config/ignition_fortress_ubuntu_jammy.yaml (hunk 2 above)
Comment (j-rivero) [reply config/ignition_fortress_ubuntu_jammy.yaml:76]: dah, I was using buggy one-liners instead of the dashboard. Sorry.

**Comment by j-rivero** [comment]
> Tools can be bumped:

[3934dba](https://github.com/ros-infrastructure/reprepro-updater/pull/166/commits/3934dbaf20c4a56b5d4e9d879c3689d84fe7e817)

File: config/ignition_fortress_ubuntu_jammy.yaml (hunk 3 above)
Comment (j-rivero) [reply config/ignition_fortress_ubuntu_jammy.yaml:77]: [725d289](https://github.com/ros-infrastructure/reprepro-updater/pull/166/commits/725d289ff5324a13b9252cdcc46ddef2284a89c5)

**Review by chapulina — APPROVED** [review]

**Pull request merged**